src/redux/actionCreators.js

```diff
@@ -191,22 +191,57 @@ export function saveGoogleDrive(file) {
   };
 }
 
-export function openGoogleDrive(file) {
+export function openGoogleDrive() {
   return dispatch => {
     dispatch({type: actType.START_OPEN_DRIVE});
-    var promise = new Promise((resolve, reject) => {
-      window.setTimeout(() => resolve(file), 1000);
-      if (!file) {
-        reject(new Error("File not opened"));
+    var scope = 'https://www.googleapis.com/auth/drive.readonly';
+    var pickerApiLoaded = false;
+    var oauthToken;
+    function onApiLoad() {
+      gapi.load('auth', {'callback': onAuthApiLoad});
+      gapi.load('picker', {'callback': onPickerApiLoad});
+    }
+    onApiLoad();
```

Comment:
looks like this is the only time `onApiLoad` gets called. Why does it need to be in a function at all?